pyproject.toml

```diff
@@ -84,6 +84,22 @@ force-exclude = '''
 [tool.codespell]
 ignore-words-list = 'astroid,crasher,asend'
 
+[tool.typos]
+default.extend-ignore-re = [
+    "(?Rm)^.*(#|//)\\s*spellchecker:disable-line$",
+]
+
+[tool.typos.default.extend-identifiers]
+IPPROTO_ND = "IPPROTO_ND"
+ERROR_INVALID_PARMETER = "ERROR_INVALID_PARMETER"
```

Comment:
I think PARMETER -> PARAMETER should be fixed, the error in cffi is in fact called `ERROR_INVALID_PARAMETER` https://www.magnumdb.com/search?q=error_invalid_parameter

I don't really understand `_windows_cffi.py`, so I don't know if that will be a backwards incompatible change, or if user code trying to handle that error is currently broken. But either case it should be fixed.

Reply:
Fixed in 9f57781.